app/graphql/resolvers/create_offer_response_resolver.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+class CreateOfferResponseResolver < BaseResolver
+  def valid?
+    return true if user?
```

Comment:
To keep it simple, this requires that a user ID is present in the JWT. Further authorization is done below.